README.md

```diff
@@ -65,78 +65,79 @@ Returns an object with all contracts from a deployment and their addresses.
 
 ## Active Deployments
 
-| Description                                            | Task ID                                                                                                 |
```

Comment:
This shouldn't have a big diff - you're doing something with your editor or line endings or something. Can you restore and edit so that the only change is the line you're adding? I can do it if you're having editor issues.

Reply:
wow, apparently my formatting is set incorrectly